src/radical/pilot/task.py

```diff
@@ -185,6 +185,7 @@ def as_dict(self):
             'tmgr':             self.tmgr.uid,
             'uid':              self.uid,
             'name':             self.name,
+            'mode':             self.mode,
```

Comment:
why do we need duplicate it? (it is in `description`)

Reply:
indeed, removed